packages/client/hmi-client/src/types/ResponsiveMatrix.ts

```diff
@@ -65,3 +65,8 @@ export type Uniforms = {
 	// cell element color data
 	uColor: Texture;
 };
+
+export enum CursorModes {
+	Select,
```

Comment:
another consistency thing, i have been using ALL_CAPS for enums (which im treating as constants).